<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>로그인</string>
  </property>
  <widget class="QLineEdit" name="idText">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius:10px;
background-color:#dedede;
}
QLineEdit:focus{
border:1.5px solid #000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwText">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius:10px;
background-color:#dedede;
}
QLineEdit:focus{
border:1.5px solid #000;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>PW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
border-radius:10px;
background-color:#f5503e;
color:#fff;
font-weight:500;
}
QPushButton:hover{
border:2px solid #fff;
}</string>
   </property>
   <property name="text">
    <string>로그인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="joinBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
border-radius:10px;
background-color:#000;
color:#fff;
font-weight:500;
}
QPushButton:hover{
border:2px solid #fff;
}</string>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:500;</string>
   </property>
   <property name="text">
    <string>아이디</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:500;</string>
   </property>
   <property name="text">
    <string>비밀번호</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
